crates/experimental-api/src/gen.rs

```diff
@@ -954,13 +964,14 @@ pub mod gen {
         pub unit: PriceUnit,
     }
 
-    #[derive(Clone, Debug, Deserialize, Serialize)]
+    #[derive(Clone, Debug, Deserialize, Serialize, Default, PartialEq)]
```

Comment:
What is a "default Felt"? Is it zero? Empty sting? It is unclear, thus a user needs to explicitly state what they need.

Reply:
Removed Default